src/Selenium2Library/keywords/_browsermanagement.py

```diff
@@ -1,6 +1,12 @@
 import os
 import robot
 from robot.errors import DataError
+try:
+    from robot.utils import DotDict
+    NO_DOT_DICT= False
+except ImportError:  # New in RF 2.9
+    NO_DOT_DICT= True
+    pass
```

Comment:
This pattern is typically better:

```python
try:
    from foo import bar
except ImportError:
    bar = None

# later:
if bar is None:
    # ...
```

In this particular case I think S2L should rather implement its own `DotDict` variant. It doesn't need to be ordered so it can be simpler than the version in RF and should also be renamed differently to avoid confusion. I already included one possible implementation of such `AttrDict` in my earlier comment to this PR.